src/vmm/src/builder.rs

```diff
@@ -145,13 +145,15 @@ impl std::convert::From<linux_loader::cmdline::Error> for StartMicrovmError {
 }
 
 #[cfg_attr(target_arch = "aarch64", allow(unused))]
+#[allow(clippy::too_many_arguments)]
 fn create_vmm_and_vcpus(
     instance_info: &InstanceInfo,
     event_manager: &mut EventManager,
     guest_memory: GuestMemoryMmap,
     uffd: Option<Uffd>,
     track_dirty_pages: bool,
     vcpu_count: u8,
+    #[cfg(target_arch = "x86_64")] seccomp_filters: BpfThreadMap,
```

Comment:
```suggestion
    #[cfg(target_arch = "x86_64")]
    seccomp_filters: BpfThreadMap,
```

Reply:
Rust's formatter seems to disagree with this one. Rust-analyzer kept moving it whenever I tried to save, and even when I turned it off, the pre-commit hook won't allow me to commit without it on the same line.